### PR TITLE
feat(topstories): Hide preference section for unsupported locales

### PR DESCRIPTION
**Pull request by csadilek**
Suggestion for a simple fix of #2973.

Reason I used "hidden" (negated showByDefault) is so nobody has to reset their prefs to keep seeing top stories.

### CODE REVIEW

File: system-addon/lib/ActivityStream.jsm [hunk 1]
```diff
@@ -25,12 +25,13 @@ const {TopStoriesFeed} = Cu.import("resource://activity-stream/lib/TopStoriesFee
...
 
 // Sections, keyed by section id
+const showTopStoriesByDefault = ["en-US", "en-CA"].includes(Services.locale.getRequestedLocale());
 const SECTIONS = new Map([
   ["topstories", {
     feed: TopStoriesFeed,
     prefTitle: "Fetches content recommendations from a configurable content provider",
      // for now, we only want to show top stories by default to the following locales
-    showByDefault: ["en-US", "en-CA"].includes(Services.locale.getRequestedLocale())
+    showByDefault: showTopStoriesByDefault
```
Comment (Mardak) [inline system-addon/lib/ActivityStream.jsm]: Keep the default inline with topstories, i.e., undo this change. So that the comment line makes sense.

File: system-addon/lib/ActivityStream.jsm [hunk 2]
```diff
@@ -84,7 +85,8 @@ const PREFS_CONFIG = new Map([
       "api_key_pref": "extensions.pocket.oAuthConsumerKey",
       "provider_name": "Pocket",
       "provider_icon": "pocket",
-      "provider_description": "pocket_feedback_body"
+      "provider_description": "pocket_feedback_body",
+      "hidden": ${!showTopStoriesByDefault}
```
Comment (Mardak) [inline system-addon/lib/ActivityStream.jsm]: Instead just do `${!SECTIONS.get("topstories").showByDefault}`

File: system-addon/lib/ActivityStream.jsm [hunk 3]
```diff
@@ -24,13 +24,14 @@ const {TopStoriesFeed} = Cu.import("resource://activity-stream/lib/TopStoriesFee
 
 const REASON_ADDON_UNINSTALL = 6;
 
+// For now, we only want to show top stories by default to the following locales
+const showTopStoriesByDefault = ["en-US", "en-CA"].includes(Services.locale.getRequestedLocale());
```
Comment (Mardak) [inline system-addon/lib/ActivityStream.jsm]: This is fine with the comment line moved up.

**Comment by as-pine-proxy** [comment]
[Treeherder: 25b950e221e91b4a1121dd4dc3d00b6cdf1c9ac8](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=25b950e221e91b4a1121dd4dc3d00b6cdf1c9ac8)
